pyaedt/edb_core/edb_data/siwave_simulation_setup_data.py

```diff
@@ -13,7 +15,7 @@ def __init__(self, parent):
     @property
     def sim_setup_info(self):
         """EDB internal simulation setup object."""
```

Comment:
It doesn't seem like this object and the next should have the same exact description. Please revise the one that isn't accurate.

Reply:
@PipKat there are two redundant methods sharing the same name. I suppose you are refering to them. They are removed.